_posts/2022-06-21-walkthrough.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Beets: A Guided Walkthrough"
+layout: main
+section: blog
+---
+
+## Overview
+This is a step by step walkthrough for getting started with the beets CLI using a sample music library. The sample music library includes public domain recordings of Debussy, Chopin, and Virginia Liston compiled from Open Music Archive, IMSLP, and Internet Archive.
+
+We will use this library in its messy/unaltered state to showcase the power of the beets CLI by correcting obscure metadata and organizing the music library with the auto-tagger (and a little manual input). Lastly, we will host the library locally using the beets web plugin to stream the audio once finished and admire our results. Let's get started!
+
+If desired, there are resources included below to help find additional recordings that are in the public domain if you'd like to grow your sample music library. Note: be sure the release you're looking to include has metadata in the MusicBrainz database for best results.
```

Comment:
Maybe it would be nice to link "MusicBrainz" to musicbrainz.org on the first mention?

Reply:
Can do!